packages/ag-charts-community/src/chart/callbacks.test.ts

```diff
@@ -191,10 +191,16 @@ describe('AG-13024 API context', () => {
         test('tooltipRenderer', async () => {
             tooltipRenderer.expect().toHaveBeenCalledTimes(0);
             await threeTooltipCallback();
-            tooltipRenderer.expect().toHaveBeenCalledTimes(3);
+            tooltipRenderer.expect().toHaveBeenCalledTimes(9);
             tooltipRenderer.expect().nthCalledWithContext(0, seriesContext0);
             tooltipRenderer.expect().nthCalledWithContext(1, seriesContext1);
             tooltipRenderer.expect().nthCalledWithContext(2, seriesContext2);
+            tooltipRenderer.expect().nthCalledWithContext(3, seriesContext1);
+            tooltipRenderer.expect().nthCalledWithContext(4, seriesContext0);
+            tooltipRenderer.expect().nthCalledWithContext(5, seriesContext2);
+            tooltipRenderer.expect().nthCalledWithContext(6, seriesContext2);
+            tooltipRenderer.expect().nthCalledWithContext(7, seriesContext0);
+            tooltipRenderer.expect().nthCalledWithContext(8, seriesContext1);
```

Comment:
While curious, this ordering is somewhat as expected. Also, the tooltips are concatenated which is why it is called this many times.